<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScene</class>
 <widget class="QWidget" name="GameScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>320</y>
     <width>181</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="giveUpBtn">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
	font: 14pt &quot;华文行楷&quot;;
    font:20px;
    color:black;
    background-color:rgb(219 , 219 , 219);
    border-radius:15px;
    height:50px;
    width:200px;
}

QPushButton:hover
{
    background-color:rgb(219 , 218 , 240);
}

QPushButton:pressed
{
    background-color:rgb(219 , 217 , 240);
    padding-left:3px;
    padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>认输</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="backBtn">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
	
	font: 14pt &quot;华文行楷&quot;;
    font:20px;
    color:black;
    background-color:rgb(219 , 219 , 219);
    border-radius:15px;
    height:50px;
    width:200px;
}

QPushButton:hover
{
    background-color:rgb(219 , 218 , 240);
}

QPushButton:pressed
{
    background-color:rgb(219 , 217 , 240);
    padding-left:3px;
    padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="repentBtn">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
	font: 14pt &quot;华文行楷&quot;;
    font:20px;
    color:black;
    background-color:rgb(219 , 219 , 219);
    border-radius:15px;
    height:50px;
    width:200px;
}

QPushButton:hover
{
    background-color:rgb(219 , 218 , 240);
}

QPushButton:pressed
{
    background-color:rgb(219 , 217 , 240);
    padding-left:3px;
    padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>悔棋</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="restart">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
	font: 14pt &quot;华文行楷&quot;;
    font:20px;
    color:black;
    background-color:rgb(219 , 219 , 219);
    border-radius:15px;
    height:50px;
    width:200px;
}

QPushButton:hover
{
    background-color:rgb(219 , 218 , 240);
}

QPushButton:pressed
{
    background-color:rgb(219 , 217 , 240);
    padding-left:3px;
    padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>重新开始</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>261</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="6" column="0">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>可悔棋数：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>未准备</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>对手：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>执子：</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer_10">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>倒计时：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>连接状态：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLable{
	font-family:&quot;华文琥珀&quot;;
}</string>
      </property>
      <property name="text">
       <string>我：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="1">
     <spacer name="verticalSpacer_11">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="1" colspan="2">
     <widget class="QLabel" name="times">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>60s</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>未连接</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>无</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>未开始</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0">
     <spacer name="verticalSpacer_12">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="1">
     <spacer name="verticalSpacer_13">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
